usr/src/uts/common/fs/zfs/arc.c

```diff
@@ -301,19 +303,23 @@ uint_t arc_reduce_dnlc_percent = 3;
 int zfs_arc_evict_batch_limit = 10;
 
 /* number of seconds before growing cache again */
-static int		arc_grow_retry = 60;
+int arc_grow_retry = 60;
 
-/* number of milliseconds before attempting a kmem-cache-reap */
-static int		arc_kmem_cache_reap_retry_ms = 1000;
+/*
+ * Minimum time between calls to arc_kmem_reap_now().  Note that this will
```

Comment:
s/now/soon/

Reply:
Thanks.  Fixed this and another instance of reap_now() in a comment.